code/modules/client/client defines.dm

```diff
@@ -94,5 +94,8 @@
 	var/list/seen_messages = list()
 	var/toggle_runechat_outlines = TRUE
 
+	// Voting & civic duty
+	var/ivoted = FALSE
```

Comment:
Technically if you're ready to grapple with how the information is stored you can have it preserve the "I voted" status

Reply:
No. This badge is meant to last for one round only.